dynamic_programming/parentheses.py

```diff
@@ -0,0 +1,66 @@
+import math
+from operator import add, sub, mul
+
+
+operators = {"+": add, "-": sub, "*": mul}
+
+
+def calculate(first_operand, second_operand, operator) -> int:
+    """
+    Evaluate the expression and return the result.
+
+    >>> calculate(2,2,"+")  # add()
+    4
+    >>> calculate(2,2,"-")  # sub()
+    0
+    >>> calculate(2,2,"*")  # mul()
+    4
+    """
+    return operators[operator](first_operand, second_operand)
+
+
+def MinandMax(maximum_numbers, minimum_numbers, i, j, operator) -> int:

```

Comment:
```suggestion
def MinandMax(maximum_numbers: List[int], minimum_numbers: List[int], i: int, j: int, operator: str) -> int:
```
Please do not mark this resolved until the function definition looks like this and has doctexts.